.bookignore

```diff
@@ -0,0 +1,7 @@
+.bookignore
+.github
+.gitignore
+.travis.yml
```

Comment:
no reason for that

Reply:
Perhaps I should have separated that, but having a single way to build, test and deploy was a good idea IMHO.

Reply:
My argument here is that there's no need to ignore that particular path, as it's not use used and you have removed that file.

Reply:
Oh good point - this was a leftover from an earlier iteration. Will update.